### DISCUSSION TITLE
Remote Desktop using v86

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#593] [Q&A]
I recently created https://github.com/hello-smile6/libv86-terminal-forwarding , how is it?

**Comment by 621625** [comment]
Cannot see the video, play button did not work for me.

**Reply by hello-smile6** [reply]
@bitcoinmeetups Try downloading it and opening with a native video player.

**Reply by Darin755** [reply]
You consider putting it in a more open media format that does not require codecs

**Reply by hello-smile6** [reply]
@Darin755 Any suggestions?

**Reply by 621625** [reply]
> @bitcoinmeetups Try downloading it and opening with a native video player.

I can't. Cannot open it and cannot download it. Maybe embed it some other way please?

**Reply by hello-smile6** [reply]
@bitcoinmeetups https://user-images.githubusercontent.com/73048226/148630248-42ad3ea7-cb1f-4a6c-833d-541bbdc12025.mp4 
Try downloading that

**Comment by Darin755** [comment]
What is this?

**Reply by hello-smile6** [reply]
@Darin755 Remote access using WebRTC
